src/halfpipe/tui/templates/selection_template.py

```diff
@@ -126,13 +125,12 @@ def compose(self) -> ComposeResult:
                 Button("Rename", variant="primary", classes="rename_button", id="rename_item_button"),
                 Button("Duplicate", variant="primary", classes="duplicate_button", id="duplicate_item_button"),
                 Button("Delete", variant="primary", classes="delete_button", id="delete_item_button"),
-                # Button("Sort", variant="primary", classes="sort_button"),
                 classes="buttons",
             ),
-            *[
-                Collapsible(title=self.ITEM_MAP[f], id="list_" + f, classes="list", collapsed=False)
-                for f in self.ITEM_MAP.keys()
-            ],
+            # *[
```

Comment:
Could you delete any outdated commented-out code before merging?

Reply:
ok